cypress/support/pages/admin/pageAdministration.js

```diff
@@ -269,5 +279,68 @@ class Management {
       .find('input')
       .click({ force: true })
   }
+
+  schoolNumberIsAdded() {
+    cy.get(Management.#schoolNumberForm)
+        .should('not.be.empty')
+  }
+
+
+  migrationButtonIsEnabled() {
```

Comment:
here as well method name can start with the verb -> seeMigrationButtonIsEnabled()